scadafence/operations.py

```diff
@@ -18,9 +18,9 @@ def __init__(self, config, *args, **kwargs):
         self.secret_key = config.get('secret_key')
         url = config.get('server_url').strip('/')
         if not url.startswith('https://') and not url.startswith('http://'):
-            self.url = 'https://{0}/api/'.format(url)
```

Comment:
Removed extra slash from the endpoint URL in operation.py file.

UTC is missing for this change.

Reply:
and impact is auth or connector configuration

Reply:
It will impact in connector configuration as well for all the actions.

UnitTest Cases:
1.> Verify check health with valid input -> Connected.
2.>  Verify check health with invalid input -> Disconnected.
3.> Verify all the actions with valid and invalid inputs.

Reply:
LGTM. Thanks adding these details, appreciate it.